acl_setup.php

```diff
@@ -174,6 +181,25 @@
  $gacl->add_object('patients', 'Lab Results (write,addonly optional)','lab', 10, 0, 'ACO');
      // xl('Lab Results (write,addonly optional)')
 
+
+ $gacl->add_object('groups', 'Add/Update groups','gadd', 10, 0, 'ACO');
+ // xl('Add update groups (read ,write)')
+
+ $gacl->add_object('groups', 'Create/Update groups appointment in calender','gcalendar', 10, 0, 'ACO');
+// xl('Using calendar add edit event with groups (read ,write)')
+
+ $gacl->add_object('groups', 'Group log','glog', 10, 0, 'ACO');
+// xl('groups encounter log (read ,write)')
+
+ $gacl->add_object('groups', 'Group detailed log of appointment in patient record','gdlog', 10, 0, 'ACO');
+// xl('groups encounter log (read ,write)')
+
+ $gacl->add_object('groups', 'Send message from the permanent group therapist to the personal therapist','gm', 10, 0, 'ACO');
+ // xl('Messages between therapists (read ,write)')
```

Comment:
also, where is the 'read' coming from. It should be 'view' and should look like this '(write, read optional)'